xbmc/utils/GLUtils.cpp

```diff
@@ -153,25 +153,25 @@ void LogGraphicsInfo()
 
   s = glGetString(GL_VENDOR);
   if (s)
-    CLog::Log(LOGINFO, "GL_VENDOR = {}", s);
+    CLog::Log(LOGINFO, "GL_VENDOR = {}", const_cast<char*>(reinterpret_cast<const char*>(s)));
```

Comment:
Maybe instead of casting the type in the log call just cast the `glGetString` calls to avoid double casts?

ie.
```
const char* s;

s = reinterpret_cast<const char*>(glGetString(GL_VENDOR));
...
```

Reply:
I agree looks tidier. Updated.